### PR TITLE
feat(konflux): onboard to konflux for deployment

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

File: config/stage_test/local/kustomization.yaml [hunk 1]
```diff
@@ -25,5 +25,5 @@ resources:
 
 images:
 - name: controller
-  newName: quay.io/cloudservices/floorist-operator
+  newName: quay.io/redhat-services-prod/insights-management-tenant/insights-floorist/floorist-operator
```
Comment (vkrizan) [inline config/stage_test/local/kustomization.yaml:28]: Is there any chance to configure these images to be public, for anyone to test this locally?

**Review by vkrizan — APPROVED** [review]

File: config/stage_test/local/kustomization.yaml (hunk 1 above)
Comment (skateman) [reply config/stage_test/local/kustomization.yaml:28]: Ask Ernesto to make them public.

Comment (skateman) [reply config/stage_test/local/kustomization.yaml:28]: EDIT: don't make this public, but request access via the link I DMed you

**Pull request merged**